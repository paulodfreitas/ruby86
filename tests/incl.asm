.module entry_code
.pseg
.org 0
        irmovl $7fffffff, %edx
        incl %edx
        incl %edx
        incl %edx
        halt
.end
