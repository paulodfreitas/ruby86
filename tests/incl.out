@ 0000
30
82
ff
ff
ff
7f
64
82
64
82
64
82
10
@ 000d
